backend/tests/wallet_tests/liquidity/client/test_client.py

```diff
@@ -9,7 +9,7 @@
 #
 #
 # def test_get_quote_for_supported_currency_pair(liquidity_provider_session):
-#     pair = CurrencyPair(Currency.Coin1, Currency.USD)
+#     pair = CurrencyPair(Currency.XUS, Currency.USD)
```

Comment:
delete file?

Reply:
I don't have opinion on this, if not useful anymore, it should be deleted, but it's better to be another PR